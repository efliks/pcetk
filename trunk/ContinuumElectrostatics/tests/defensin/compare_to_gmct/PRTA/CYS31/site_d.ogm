ON_GEOM_CENT 121 2.000000
(28.761640 32.935948 13.760486) 101 1.000000
(28.761640 32.935948 13.760486) 101 0.500000
(28.761640 32.935948 13.760486) 101 0.250000
