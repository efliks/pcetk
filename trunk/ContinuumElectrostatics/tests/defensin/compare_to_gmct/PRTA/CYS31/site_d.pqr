ATOM      0  CB  CYS    31      29.225  33.108  13.148  -0.20000   1.700      0      0      0      0
ATOM      0  HB1 CYS    31      29.943  32.405  13.579  -0.05000   1.000      0      0      0      0
ATOM      0  HB2 CYS    31      29.389  34.057  13.714  -0.05000   1.000      0      0      0      0
ATOM      0  SG  CYS    31      27.549  32.538  13.522  -0.70000   1.800      0      0      0      0
ATOM      0  HG1 CYS    31      27.703  32.572  14.839  -0.00000   0.300      0      0      0      0
