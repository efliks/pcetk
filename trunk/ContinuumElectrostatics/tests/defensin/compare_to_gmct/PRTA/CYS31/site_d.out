Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named scratch/PRTA/CYS31/site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (24.1776, 30.1496, 6.10909)) 121 2
Finer level:    (28.7616, 32.9359, 13.7605) 101 1
Finer level:    (28.7616, 32.9359, 13.7605) 101 0.5
Finest level:   (28.7616, 32.9359, 13.7605) 101 0.25


Self energy of scratch/PRTA/CYS31/site_d in three dielectric environment = 150.159
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/CYS31/site_d with background charge set in three dielectric environment = -1.29704
(probably in kcal/mole)



Interaction energies of scratch/PRTA/CYS31/site_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.0154035
0   1 : 0.215369
1   0 : 1.60512
1   1 : 11.7629
2   0 : -0.0125822
2   1 : 0.301366
3   0 : 0.0448941
3   1 : 0.344561
4   0 : -0.384534
4   1 : -0.0374597
5   0 : -0.079353
5   1 : 0.930459
6   0 : -0.0147184
6   1 : 0.926121
7   0 : -0.212275
7   1 : -0.00424367
8   0 : -0.156141
8   1 : -0.0126893
9   0 : -0.123868
9   1 : 1.26268
10   0 : 0.00376765
10   1 : 0.390978
11   0 : 0.00569444
11   1 : 0.0517567
12   0 : -0.0186265
12   1 : -0.000448801
13   0 : 0.0247868
13   1 : 1.98977
14   0 : 92.6162
14   1 : 300.316
Total runtime: 9.44 s user 0.06 s system 
